storage/client/queue/src/main/java/com/azure/storage/queue/QueueAsyncClient.java

```diff
@@ -93,7 +93,7 @@ public URL getQueueUrl() {
         try {
             return new URL(client.url());
         } catch (MalformedURLException ex) {
-            LOGGER.asError().log("Queue URL is malformed");
+            logger.asError().log("Queue URL is malformed");
```

Comment:
@JonathanGiles, based on the logging specification should a warning be logged? This is recoverable.

Reply:
Correct. Error is for unrecoverable situations, warning is recoverable.

Reply:
Done.